.gitignore

```diff
@@ -1,27 +1,24 @@
 # Ignore files generated by R
+# jekyll built site
+_site
+.DS_Store
 .RData
 .Rhistory
-.DS_Store
-data/.DS_Store
-differential-expression/.DS_Store
-
-# Ignore ID refinery mappings for irrelevant platforms
-compare-processing/data/id_refinery/all_1536267482.zip
 
 # This data is downloaded from another part of the repo in essence
-normalize-own-data/data/GSE39842_RMA_converted.tsv
```

Comment:
Looks like this file also needs to be removed from this file path (forgive me if I am wrong, I checked out your branch and still see the file at this path but it is quite possible that it is just local).

Reply:
I think I fixed this?

Reply:
Looks like it 👍